en/data_links/sik_radio.md

```diff
@@ -7,61 +7,235 @@ Information about *using* SiK Radio can be found it the *PX4 User Guide*: [Telem
 The ("developer") information below explains how to build SiK firmware from source and configure it using AT commands.
 
 
-## Build Instructions
-
-You will need to install the required 8051 compiler, as this is not included in the default PX4 Build toolchain.
-
-### Mac OS
-
-Install the toolchain:
-
-```sh
-brew install sdcc
-```
-
-Build the image for the standard SiK Radio / 3DR Radio:
-
-```sh
-git clone https://github.com/LorenzMeier/SiK.git
-cd SiK/Firmware
-make install
-```
-
-Upload it to the radio \(**change the serial port name**\):
-
-```
-tools/uploader.py --port /dev/tty.usbserial-CHANGETHIS dst/radio~hm_trp.ihx
-```
-
-## Configuration Instructions
-
-The radio supports AT commands for configuration.
-
-```sh
-screen /dev/tty.usbserial-CHANGETHIS 57600 8N1
-```
-
-Then start command mode:
-
-> **Note** DO NOT TYPE ANYTHING ONE SECOND BEFORE AND AFTER
-
-```
-+++
-```
-
-List the current settings:
-
-```
-ATI5
-```
-
-Then set the net ID, write settings and reboot radio:
-
-```
-ATS3=55
-AT&W
-ATZ
-```
-
-> **Note** You might have to power-cycle the radio to connect it to the 2nd radio.
-
+## Supported Radio Hardware
+
+The SiK repository includes bootloaders and firmware for the following telemetry radios (25 FEB 2020):
+- HopeRF HM-TRP
+- HopeRF RF50-DEMO
+- RFD900
+- RFD900a
+- RFD900p
+- RFD900pe
+- RFD900u
+- RFD900ue
+
+**SiK does not currently support RFD900x or RFD900ux telemetry radios**.
```

Comment:
So are you saying that you have to use non-SiK firmware, or that you can't build your own SiK firmware (ie is the RFDesign software "non-Sik" or is is just prebuilt out of band?

Reply:
Good catch -- to clarify, I believe the RFD900x/ux still uses SiK firmware (they can be queried via AT commands, just like the other radios), but these radios are not currently supported by the tools and firmware that is packaged by default in the linked SiK repository.

There is a directory (SiK/Firmware/dst) that contains firmware for the older models (.ihx files), but does not contain firmware the RFD900x/ux (which, for whatever reason, RFDesign has chosen to release as .bin files).  The newer firmware can be downloaded here: http://files.rfdesign.com.au/firmware/

Reply:
OK, I've created https://github.com/PX4/Devguide/pull/966 to get just this bit in for now. Hopefully Lorenz will provide advice on the rest at some point. 

Thanks again for all your help.